src/content/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/version-trends-compare-user-adoption-metrics-performance.mdx

```diff
@@ -11,65 +11,42 @@ redirects:
-4. Optional: To configure email settings, see [Email reports](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-monitoring-email-notifications#version-trends).
+To view the version trends report go to **[one.newrelic.com](https://one.newrelic.com) > Mobile > (select an app) > App > Version trends**.
 
-## View version trend details [#details]
+![A screenshot of the version trends UI that shows several app versions and the performance differences as percentages between them](./images/version-trends-ui.png "Mobile version trends UI")
 
-Each version includes several metrics to analyze performance, user adoption, and key technical indicators.
-
-<CollapserGroup>
-  <Collapser
-    id="performance"
-    title="Performance across releases"
-  >
-    An important feature is comparison of app performance across releases, based on data collected when each version was at its peak of popularity. You can select four different views of data for when each version was in production, including:
+Each new version of your mobile app is compared against the previous version. For a selected time period, the version trends report compares the performance increases and decreases of your app versions against the previous version.
 
-    * Launch day (default)
-    * Launch week
-    * Launch month
-    * All available data for the version
+For example, use the timepicker dropdown to select default or custom time periods to query:
 
-    This quickly helps you see whether the adoption and performance characteristics of an app's version are better or worse than others.
+![This is a screenshot of the time picker within the version trends UI. Dates selected are March 24th through March 25th.](./images/version-trends-time-picker.png "Mobile version trends time picker")
 
-    Summary mobile app information for each release includes:
+## Understanding version trend details [#details]
 
-    * **Versions:** Versions of your app release, sorted by date of highest popularity
-    * **Sessions:** Count of mobile app sessions showing usage of each release
-  </Collapser>
+Each version includes several metrics to analyze performance, user adoption, and key technical indicators.
 
+<CollapserGroup>
   <Collapser
     id="adoption"
     title="Critical user adoption metrics"
   >
-    You can compare different versions to see how successfully they have been adopted by users.
-
-    * **Total users:** Count of unique users (devices) active
-    * **Upgrades:** Count of unique users (devices) active that were upgrades from a previous app version
-    * **New installs:** Count of unique users (devices) active who performed a new install of this app
+    You can compare different versions to see how successfully they've been adopted by your users.
 
-    Upgrades and new installs combine to reveal adoption in the form of total unique installs.
+    * **App launches:** Count of the number of times the application has been launched by a user
```

Comment:
This section uses initial cap after  colon but the next section starting at line 58 uses l/c. I think Microsoft Manual of Style says l/c after colon. All I really care about is consistency! :-)